sstmac/software/launch/app_launcher.cc

```diff
@@ -48,10 +48,12 @@ Questions? Contact sst-macro-help@sandia.gov
 #include <sstmac/software/process/operating_system.h>
 #include <sstmac/software/process/app.h>
 #include <sstmac/common/thread_lock.h>
+#include <sstmac/libraries/nlohmann/json.hpp>
 #include <sprockit/sim_parameters.h>
 #include <sprockit/util.h>
 #include <unistd.h>
 #include <getopt.h>
+#include <sstream>
```

Comment:
Can remove `sstream` if you use `json::parse`

Reply:
Removed sstream include in cs 5841f71.